admin/index_m.html

```diff
@@ -1513,7 +1523,12 @@
                     </div>
                 </div>
             </div>
-            <div id="tab-smart-enums"  class="col s12 page">
+            <div id="tab-smart-enums" class="col s12 page">
+                <div class="row">
+                    <div class="col s6">
+                        <img src="cloud.png" class="logo">
```

Comment:
Is there a reason you're introducing the cloud logo on every page? It causes a very ugly grey bar. Especially in the smart devices tab:

![image](https://user-images.githubusercontent.com/5904171/35780512-b5fe2bdc-09dc-11e8-8524-e1845ae990a1.png)